src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -282,10 +283,11 @@ public function prepend($path, $data, $separator = PHP_EOL)
     /**
      * Append to a file.
      *
-     * @param  string  $path
-     * @param  string  $data
-     * @param  string  $separator
+     * @param  string $path
+     * @param  string $data
+     * @param  string $separator
```

Comment:
You're going to need to revert all these param space changes. There should be 2 spaces before and after the param type.